Tests/SourceKitLSPTests/RenameTests.swift

```diff
@@ -486,4 +462,182 @@ final class RenameTests: XCTestCase {
         """
     )
   }
+
+  func testCrossFileSwiftRename() async throws {
+    let ws = try await SwiftPMTestWorkspace(
+      files: [
+        "a.swift": """
+        func 1️⃣foo2️⃣() {}
+        """,
+        "b.swift": """
+        func test() {
+          3️⃣foo4️⃣()
+        }
+        """,
+      ],
+      build: true
+    )
+
+    let (aUri, aPositions) = try ws.openDocument("a.swift")
+    let response = try await ws.testClient.send(
+      RenameRequest(textDocument: TextDocumentIdentifier(aUri), position: aPositions["1️⃣"], newName: "bar")
+    )
+    let changes = try XCTUnwrap(response?.changes)
```

Comment:
Could we just apply the edits as in the single file rename? Or if not that, at least have a helper for creating a `TextEdit` from a file + name + two markers?